tests/functional/kata-agent-apis/api-tests/test_vm_GuestDetails.bats

```diff
@@ -9,15 +9,17 @@ load "${BATS_TEST_DIRNAME}/../setup_common.sh"
 
 setup_file() {
     info "setup"
-    sudo rm qmp.sock console.sock || echo "No existing qmp.sock/console.sock"
 }
 
 @test "Test GetGuestDetails: Boot qemu pod vm and run GetGuestDetails" {
     info "Boot qemu vm, establish connection with agent inside the vm and send GetGuestDetails command"
+    local test_dir=$(mktemp -d)
+    pushd $test_dir
     local cmds=()
     cmds+=("--vm qemu -c GetGuestDetails")
     run_agent_ctl "${cmds[@]}"
-    sudo rm qmp.sock console.sock
+    popd
+    rm -rf $test_dir
```

Comment:
This is definitely a better approach. Just one question : why is `sudo` not required anymore ?

Reply:
@gkurz considering a dir under `/tmp` when running the tests, I assumed the same could be deleted without elevated privileges.
Please let me know otherwise and I will update.